source/xml/reading.md

```diff
@@ -376,6 +377,7 @@ their respective XML elements:
             }
             return $books;
         },
+        // handle a single book
         '{http://example.org/books}book' => function($reader) {
             $book = new Book();
```

Comment:
But it kind of touched upon the same issue.. how do we pass configuration?

Aside from that I also think that it would probably be smart to add a KeyValue deserializer that ignores namespaces. Great for your use-case.